airflow/api_fastapi/core_api/datamodels/task_instances.py

```diff
@@ -150,3 +150,11 @@ class TaskInstanceHistoryCollectionResponse(BaseModel):
 
     task_instances: list[TaskInstanceHistoryResponse]
     total_entries: int
+
+
+# Response Models
+class TaskInstancesLogResponseObject(BaseModel):
```

Comment:
```suggestion
class TaskInstancesLogResponse(BaseModel):
```

Making it consistent with other classes

Reply:
Updated it accordingly, PTAL.